src/typings/sharify.d.ts

```diff
@@ -25,7 +25,7 @@ declare module "sharify" {
       CURRENT_USER: User
       EIGEN: boolean
       readonly ENABLE_FAIR_PAGE_EXHIBITORS_TAB: boolean
-      readonly ENABLE_SAVED_SEARCH: boolean
+      ENABLE_SAVED_SEARCH: boolean
```

Comment:
**question**: Are we able to keep this variable readonly? Maybe by mocking sharify in the specs?

Reply:
If the alert pages will only be accessible via a direct link `users/alerts`, then we don't need to delete `readonly` for tests.